third_party/xla/xla/layout.h

```diff
@@ -39,30 +39,38 @@ class Shape;
 class Tile {
  public:
   Tile() = default;
+  
   explicit Tile(absl::Span<const int64_t> dimensions)
       : dimensions_(dimensions.begin(), dimensions.end()) {}
 
   // De/Serialize a Tile to and from a TileProto.
   static Tile CreateFromProto(const TileProto& tile_proto) {
     return Tile(tile_proto.dimensions());
   }
+  
   TileProto ToProto() const;
   void SetProto(TileProto& tile_proto) const;
 
   bool operator==(const Tile& other) const {
     return dimensions() == other.dimensions();
   }
-  bool operator!=(const Tile& other) const { return !(*this == other); }
 
-  void Print(Printer* printer) const;
+  bool operator!=(const Tile& other) const {
+    return !(*this == other);
+  }
```

Comment:
There is no change here other than layout. We use automatic formatters, so this should not be changed.